src/imitation/algorithms/adversarial.py

```diff
@@ -243,35 +264,41 @@ def train_gen(self, total_timesteps: Optional[int] = None, callback=None):
 
     with logger.accumulate_means("gen"):
       self.gen_policy.set_env(self.venv_train_norm_buffering)
-      # TODO(adam): learn was not intended to be called for each training batch
```

Comment:
if we're removing this warning comment, feel like we should also point to the fixed branch of stable baselines in dependency (or just wait until it gets merged to master)

Reply:
I was just thinking about making setup.py point to the Runner patch (had an error earlier this week where I installed SB without my patch).

Reply:
Looks like it's fine for us to go ahead and merge the Runner patch after your review+approval. Let's merge this PR after that change.